WeakAuras/Init.lua

```diff
@@ -373,9 +374,26 @@ local flavor = flavorFromTocToNumber[flavorFromToc]
 if versionStringFromToc == "@project-version@" then
   versionStringFromToc = "Dev"
   buildTime = "Dev"
+  WeakAuras.buildType = "dev"
 end
 --@end-debug@
 
+--[[@experimental@
+WeakAuras.buildType = "pr"
+--@end-experimental@]]
+
+--[[@alpha@
+WeakAuras.buildType = "alpha"
+--@end-alpha]]
+
+--[[@non-alpha@
```

Comment:
Are PR builds also alpha builds?

Reply:
I ended up adding a @non-experimental@ flag that the build actions check for, to avoid that problem.